libp2p/Capability.cpp

```diff
@@ -28,9 +28,10 @@ using namespace std;
 using namespace dev;
 using namespace dev::p2p;
 
-Capability::Capability(std::shared_ptr<Session> _s, HostCapabilityFace* _h, unsigned _idOffset): m_session(_s), m_hostCap(_h), m_idOffset(_idOffset)
+Capability::Capability(std::shared_ptr<Session> _s, HostCapabilityFace* _h, unsigned _idOffset, uint16_t _protocolID):
+	m_session(_s), m_hostCap(_h), m_idOffset(_idOffset), c_protocolID(_protocolID)
```

Comment:
"Premature optimization is the root of all evil."
We will refactor the code when it will be necessary.